modules/cluster-logging-forwarding-json-logs-to-the-default-elasticsearch.adoc

```diff
@@ -0,0 +1,48 @@
+[id="cluster-logging-forwarding-json-logs-to-the-default-elasticsearch_{context}"]
+= Forwarding JSON logs to the default Elasticsearch log store
+
+For the default Elasticsearch log store that OpenShift Logging manages, you must create a different index for each format in advance if your JSON log entries _have different formats_. Otherwise, forwarding different formats to the same index can cause type conflicts and cardinality problems.
+
+.Procedure
+
+. Add the following snippet to your `ClusterLogForwarder` CR YAML file.
++
+[source,yaml]
+----
+outputDefaults:
+- elasticsearch:
+    structuredTypeKey: <log record field>
+    structuredTypeName: <name>
+pipelines:
+- inputRefs:
+  - application
+  outputRefs: default
+  parse: json <4>
```

Comment:
Callout number with no text below.  Do you want to remove this?  (Also not sure why it's set to 4?)

Reply:
Updated. Thanks.